cmds/lddfiles/lddfiles.go

```diff
@@ -46,6 +46,11 @@ func ldd(o io.Writer, s ...string) error {
 }
 
 func main() {
+
+	if len(os.Args[1:]) < 1 {
```

Comment:
This is not needed. Try it ... because go ranges are half open, the for loop never runs if there are no args. 
This is possibly confusing but if you are calling this program there are cases where it might be called with no args and returning a zero length list is reasonable.